README.adoc

```diff
@@ -2723,9 +2723,9 @@ def some_method_with_parameters(param1, param2)
 end
 ----
 
-=== Method Invocation Parentheses [[method-invocation-parens]]
+=== Method Call Parentheses [[method-invocation-parens]][[method-call-parens]]
```

Comment:
Kept old anchors.

AsciiDoc reference for additional anchors https://docs.asciidoctor.org/asciidoc/latest/sections/custom-ids/#assign-additional-ids